src/Form/Extension/Field/Type/FormTypeFieldExtension.php

```diff
@@ -229,11 +229,7 @@ public function getValueFromFieldDescription($object, FieldDescriptionInterface
     protected function getClass(FormBuilderInterface $formBuilder)
     {
         foreach ($this->getTypes($formBuilder) as $type) {
-            if (!method_exists($type, 'getName')) { // SF3.0+
-                $name = get_class($type);
-            } else {
-                $name = $type->getName();
-            }
+            $name = get_class($type);
```

Comment:
Not sure about this change. In theory it will be better to always use FQCN,  but what happens if someone is still using 2.8 with old getName function?

Reply:
If they do that, their class still has an FQCN, right? I don't understand what this method is supposed to return though. Do you?

Reply:
I have no idea

Reply:
Should I revert this one?

Reply:
I dunno, but when in doubt, it's safer to revert it.

Reply:
Reverted, but with a better message